test/Microsoft.AspNet.Mvc.FunctionalTests/WebApiCompatShimBasicTest.cs

```diff
@@ -61,24 +61,21 @@ public class WebApiCompatShimBasicTest
                 content);
         }
 
-#if !DNXCORE50
```

Comment:
Just happened to notice this test could be enabled with Core-specific expectations.